documentation/src/main/asciidoc/stories/assembly_externalizing_spring_session.adoc

```diff
@@ -0,0 +1,10 @@
+[id='externalizing-spring-session']
+:context: spring-session
+
+= Externalizing sessions with Spring Session
+
+include::{topics}/proc_externalizing_spring_session.adoc[leveloffset=+1]
```

Comment:
Being a real stickler with this one but please mv to a new file: `proc_externalizing_sessions_spring.adoc`